github-repo-manager/src/main/java/com/axonivy/github/file/GitHubMissingFiles.java

```diff
@@ -0,0 +1,35 @@
+package com.axonivy.github.file;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.axonivy.github.file.GitHubMissingFilesDetector.FileMeta;
+
+public class GitHubMissingFiles {
+
+  private static final List<String> WORKING_REPOS = List.of("axonivy", "axonivy-market");
+  private static final FileMeta LICENSE = new FileMeta("LICENSE", "Add license", "Add_License",
+      "Add Apache 2.0 license");
+  private static final FileMeta SECURITY = new FileMeta("SECURITY.md", "Add security file", "Add_Security",
+      "Add Security.md file to repo");
+  private static final FileMeta CODE_OF_CONDUCT = new FileMeta("CODE_OF_CONDUCT.md", "Add code of conduct file",
```

Comment:
create a constant for List.of(LICENSE, SECURITY, CODE_OF_CONDUCT); 
the iterate over the items in the list in the main method.